application/controllers/teams_controller.py

```diff
@@ -44,6 +47,9 @@ def index(self, params):  # pylint: disable=W0613
     def create(self, params):
         """Logic for creating a team"""
         team = Team(name=params["name"]).save()
+        print(team)
+        self.notify_of_team_creation(team)
+        #spawn_n(self.notify_of_team_creation, team)
```

Comment:
remove? :D or maybe just uncomment if you want to make this threaded now, see other comments